crates/vhost-user-backend/tests/vhost-user-server.rs

```diff
@@ -166,13 +166,13 @@ fn vhost_user_client(path: &Path, barrier: Arc<Barrier>) {
     let addr = mem.get_host_address(GuestAddress(0x100000)).unwrap() as u64;
     let reg = mem.find_region(GuestAddress(0x100000)).unwrap();
     let fd = reg.file_offset().unwrap();
-    let regions = [VhostUserMemoryRegionInfo {
-        guest_phys_addr: 0x100000,
-        memory_size: 0x100000,
-        userspace_addr: addr,
-        mmap_offset: 0,
-        mmap_handle: fd.file().as_raw_fd(),
-    }];
+    let regions = [VhostUserMemoryRegionInfo::new(
+        0x100000,
+        0x100000,
+        addr,
+        0,
+        fd.file().as_raw_fd(),
+    )];
```

Comment:
rust-analyzer provides the context as code lenses anyway... Might leave people guessing the context of the magic numbers a bit more when not using rust-analyzer, but I guess if it simplifies the code thats ok I guess.

Reply:
> rust-analyzer provides the context as code lenses anyway... Might leave people guessing the context of the magic numbers a bit more when not using rust-analyzer, but I guess if it simplifies the code thats ok I guess.

And this was also part of the cleanups series that got meged :)